Documentation/ceph-cluster-crd.md

```diff
@@ -1382,6 +1382,7 @@ The `security` section contains settings related to encryption of the cluster.
   * `kms`: Key Management System settings
     * `connectionDetails`: the list of parameters representing kms connection details
     * `tokenSecretName`: the name of the Kubernetes Secret containing the kms authentication token
+    * `csiConnectionDetailsCMName`: the name of the Config Map containing the connection details as well as the token to use encrypted PVs from Ceph-CSI
```

Comment:
Instead of asking for a name in the CR, can we just have a constant name? For example, the csi driver already uses a constant name for a CM with the mon configuration. Or why does this need to be configurable?

Reply:
To follow up from our discussion, I still wouldn't want the CM name in the cluster CR, perhaps a bool if anything. More comments below.